lib/generators/komponent/install_generator.rb

```diff
@@ -49,6 +44,14 @@ def install_stimulus
 
       private
 
+      def stimulus_application_template
+        <<~HEREDOC
```

Comment:
Why did you replace `<<-eos` with `<<~HEREDOC`?

Reply:
Oh, nice, I did not know the squiggly heredoc :+1: https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc